config/routes.rb

```diff
@@ -0,0 +1,11 @@
+
+
+Rails.application.routes.draw do
+  resources :users
+  resources :likes
+  resources :posts
+  resources :comments
```

Comment:
- [ ] Consider when run `rails routes` in console your routes to be something like this. Kindly use nested routes to achieve this.
> ![image](https://user-images.githubusercontent.com/53402600/187722683-426c4817-a902-412a-b412-abe3de2686f5.png)